.github/workflows/ci.yml

```diff
@@ -13,4 +13,5 @@ on:
 
 jobs:
   call-ci:
-    uses: phoenix-rtos/phoenix-rtos-project/.github/workflows/ci-project.yml@master
+    uses: phoenix-rtos/phoenix-rtos-project/.github/workflows/ci-project.yml@nalajcie/ci_test
```

Comment:
I guess this should be changed before the merge

Reply:
yep, that's why I have this commit on top:
![image](https://github.com/phoenix-rtos/phoenix-rtos-project/assets/3939983/40ddef42-1397-4f62-b11b-c6f6da121342)

I'll update the PR now as I know the CI passes (I hope the CI would still pass as it would take the workflow definition from "future" merge commit but I'm not sure it works that way)